flow-typed/declarations.js

```diff
@@ -28,6 +29,8 @@ declare type ReduxAction = () => {
 
 declare type NetworkType = string
 
+declare type NetworkLabelTypes = $Values<typeof NETWORK_LABELS>
```

Comment:
hmm is this actually working? thinking if this should just be `MAIN_NETWORK_LABEL | TEST_NETWORK_LABEL`

Reply:
When using this syntax initially there was previously an error that seems to have disappeared ✨ thanks @evgenyboxer